src/extension.ts

```diff
@@ -26,7 +26,7 @@ import {createStatusBarItem} from './Config/ConfigStatusBar';
 import {CodelensProvider} from './Editor/CodelensProvider';
 import {HoverProvider} from './Editor/HoverProvider';
 import {Jsontracer} from './Jsontracer';
-import {OneExplorer} from './OneExplorer';
+import {handleRunOnecc, OneExplorer, OneNode} from './OneExplorer';
```

Comment:
I don't think that exposing handleRunOnecc or OneNode is good. You can move the command registration inside OneExplorer instead.

Reply:
Please check this out : 008bf842aba34205196cd934c79b2b5de3134614

Reply:
Good suggestion. Thanks.